README.md

```diff
@@ -35,8 +35,11 @@ mv Linux zImage
 mv Ramdisk rootfs.cpio.gz
 ```
 
-Now we can replace some of these files as required with our modifications, and
-build the FIT image and `.frm` file as described in the
+Now we can replace some of these files as required with our modifications. In 
+order to modify the rootfs, is is necessary to extract it and re-pack it as 
+indicated 
+[`below`](https://github.com/daniestevez/pluto-firmware-modifications/edit/main/README.md#extraction-and-re-packing-of-the-rootfs)
```

Comment:
This link points to the "edit" page. The correct link would be https://github.com/daniestevez/pluto-firmware-modifications/blob/main/README.md#extraction-of-the-pluto-firmware-image

I don't know if it's possible to include a relative link to a section, as in `[below](#extraction-and-re-packing-of-the-rootfs)`.